content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -229,7 +229,7 @@ constructor, or just one method:
 [source,groovy]
 ----
 package org.foo
-class Utilities {
+class Utilities implements Serializable {
```

Comment:
Hm, maybe not.  The other is static so it doesn't need to be serializable. 
On the other hand, the distinction seems like something that would be easy to mess up or forget if one goes from only having a static method to having some data.  It is worth highlighting with a warning.